plugins/rtmp-services/data/services.json

```diff
@@ -2706,8 +2706,14 @@
             ],
             "recommended": {
                 "keyint": 1,
-                "profile": "high",
-                "bframes": 0
+                "bframes": 0,
+                "max fps": 60,
+                "max video bitrate": 8000,
+                "max audio bitrate": 192,
+                "x264opts": "tune=zerolatency scenecut=0",
```

Comment:
Are you *really really really* sure you want to force `tune=zerolatency` onto your users?
It significantly reduces video quality while not reducing the overall stream latency by that much (considering you're still streaming via RTMP).

Reply:
Thanks for your message.
We will consider more detailly and change the setting later.

Reply:
hi @gxalpha,  thanks for your advice.   We have talked about the quality risk with "zerolatency".  But for CHZZK users, low latency is much more important.  So we decide to use it, thanks a lot~

Reply:
We explained the tradeoffs, and they have insisted. Resolving.